### DISCUSSION TITLE
如何让两个不同国家的人进行mc联机？

### DISCUSSION THREAD

**Discussion by opquxing** [natfrp/wiki#267] [使用问题]
mc

**Comment by zymooll** [comment]
该怎么连就怎么连

**Comment by Cassy-Lee** [comment]
一样的方式，除非节点被GFW反向墙了

**Comment by rikka0w0** [comment]
https://www.curseforge.com/minecraft/mc-mods/wsmc
给自己打波广告，国外线路优化可以白嫖cloudflare，用我的mod让mc支持websocket即可

**Reply by Cassy-Lee** [reply]
鉴定为只支持1.20的mod形式的tcp2ws，想法挺好但是不如使用其他隧道工具，还可以选择使用wss
